packages/runtime/container-runtime/src/summary/summaryFormats.md

```diff
@@ -0,0 +1,150 @@
+# Summary and snapshot formats
+
+## Introduction
+
+This document describes the format of summaries and snapshots. This should help developers understand how to look at the data, how to make changes, etc. It should also help users understand the format of their data in summary / snapshot.
+
+For difference between summaries and snapshots, see [this doc](https://fluidframework.com/docs/faq/#whats-the-difference-between-a-summary-and-a-snapshot)
```

Comment:
For learning about the differences between a "summary" and a "snapshot", refer to the FAQs section [here](https://fluidframework.com/docs/faq/#whats-the-difference-between-a-summary-and-a-snapshot)